content/docs/addons-shallow-renderer.md

```diff
@@ -30,12 +30,12 @@ function MyComponent() {
 }
 ```
 
-Then you can assert:
+你可以使用斷言：
 
 ```javascript
 import ShallowRenderer from 'react-test-renderer/shallow';
 
-// in your test:
+// 測試程式:
```

Comment:
You don't need to translate comment, thank you.

Reply:
ok.

Reply:
done